schemas/content/directory-page.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/content/directory-page",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Directory Page Object",
+  "type": "object",
+  "description": "Directory page object contains paging information for related directory when directory result is paginated.",
+  "definitions": {
+    "page": {
+      "properties": {
+        "@type": {
+          "type": "string",
+          "format": "uri",
+          "const": "https://ns.adobe.com/xdm/content/directory-page"
+        },
+        "xdm:orderBy": {
```

Comment:
I noticed that directory.schema.json does not use "xdm:" prefix, should that be used there as well?